python-avd/pyavd/_cv/workflows/finalize_workspace_on_cv.py

```diff
@@ -87,3 +91,50 @@ async def finalize_workspace_on_cv(workspace: CVWorkspace, cv_client: CVClient)
         return
 
     return
+
+
+async def process_workspace_build_errors(
+    workspace_id: str,
+    build_id: str,
+    cv_client: CVClient,
+) -> list[CVWSBuildError]:
```

Comment:
This is returning a list of dicts. Please enable pyright (part of pylance) in your repo to assist with type checking.

Reply:
Converted this (commit 75c4de28a) to fully use pre-defined models